jsonforms-editor/src/palette-panel/components/PalletePanel.tsx

```diff
@@ -87,48 +119,91 @@ export const PalettePanel = () => {
     }
   };
 
-  const classes = useStyles();
-
   return (
-    <div className={classes.palettePanel}>
-      <Tabs value={selectedTab} onChange={handleTabChange}>
-        <Tab label='Palette' data-cy='palette-tab' />
-        <Tab label='JSON Schema' data-cy='schema-tab' />
-        <Tab label='UI Schema' data-cy='uischema-tab' />
-      </Tabs>
-      <TabContent index={0} currentIndex={selectedTab}>
-        <UIElementsTree
-          className={classes.uiElementsTree}
-          elements={paletteService.getPaletteElements()}
-        />
-        <SchemaTreeView schema={schema} />
-      </TabContent>
-      <TabContent index={1} currentIndex={selectedTab}>
-        <SchemaJson
-          title='JSON Schema'
-          schema={toText(exportSchema)}
-          debugSchema={
-            schema && showDebugSchema
-              ? toText(toPrintableObject(schema))
-              : undefined
-          }
-          type='JSON Schema'
-          updateSchema={handleSchemaUpdate}
-        />
-      </TabContent>
-      <TabContent index={2} currentIndex={selectedTab}>
-        <SchemaJson
-          title='UI Schema'
-          schema={toText(exportUiSchema)}
-          debugSchema={
-            uiSchema && showDebugSchema
-              ? toText(buildDebugUISchema(uiSchema))
-              : undefined
-          }
-          type='UI Schema'
-          updateSchema={handleUiSchemaUpdate}
-        />
-      </TabContent>
-    </div>
+    <Grid container direction={'row'} wrap={'nowrap'}>
+      <Grid item className={classes.menu}>
+        <List>
+          <ListItem
+            button
+            onClick={() => handleTabChange(0)}
+            selected={selectedTab === 0}
+          >
+            <ListItemIcon data-cy='palette-tab'>
+              <BallotIcon />
+            </ListItemIcon>
+          </ListItem>
+          <ListItem
+            button
+            onClick={() => handleTabChange(1)}
+            selected={selectedTab === 1}
+          >
```

Comment:
Maybe we can encapsulate the index handling into a reusable component, similar to the other panels?